gnome-initial-setup/pages/account/gis-account-page.c

```diff
@@ -1403,6 +1403,9 @@ gis_account_page_init (GisAccountPage *page)
 void
 gis_prepare_account_page (GisDriver *driver)
 {
+  if (gis_driver_is_skipped_page (driver, PAGE_ID))
+    return;
```

Comment:
That sounds like a bug in the dots indicator, but my more general point is that the same code that calls `gjs_prepare_*_page()` could do that check, instead of sprinkling it in every page.
It also feels wrong to me that the branding page dictates behavior of the other pages; pages should be independent of each other.

An alternative approach that I prefer would be to remove the skip list from the branding keyfile, and ship the regular skip file together with the branding page keyfile configuration.